clients/web/src/views/body/UsersView.js

```diff
@@ -60,14 +59,27 @@ var UsersView = View.extend({
             parentView: this
         }).on('g:resultClicked', this._gotoUser, this);
 
+        if (getCurrentUser() && getCurrentUser().get('admin')) {
+            const userCountPromise = UserCollection.getTotalCount()
+                .done((count) => {
+                    this.usersCount = count;
+                });
+            promiseArray.push(userCountPromise);
+        }
         this.register = settings.dialog === 'register' && getCurrentUser() &&
                         getCurrentUser().get('admin');
+
+        $.when(...promiseArray)
+            .done(() => {
+                this.render();
+            });
     },
 
     render: function () {
         this.$el.html(UserListTemplate({
             users: this.collection.toArray(),
             getCurrentUser: getCurrentUser,
```

Comment:
It'd be cleaner if we made this template just take the `currentUser`, instead of a function that has to be evaluated:
```
    currentUser: getCurrentUser(),
```